src/Service.Tests/SqlTests/GraphQLSupportedTypesTests/GraphQLSupportedTypesTestsBase.cs

```diff
@@ -509,7 +539,67 @@ private static void CompareDateTimeResults(string actual, string expected)
+                // Adjusting to universal, since DateTime doesn't account for TimeZone
+                DateTime expectedDateTimeUniversal = DateTime.Parse(expectedDateTime, CultureInfo.InvariantCulture, DateTimeStyles.AdjustToUniversal);
+                DateTime actualDateTimeUniversal = DateTime.Parse(actualDateTime, CultureInfo.InvariantCulture, DateTimeStyles.AdjustToUniversal);
+                Assert.AreEqual(expectedDateTimeUniversal, actualDateTimeUniversal);
+            }
+        }
+
+        /// <summary>
+        /// Required due to different format between mysql datetimeoffset and HotChocolate datetime
+        /// result
+        /// </summary>
+        private static void CompareDateTimeOffsetResults(string actual, string expected)
+        {
+            string fieldName = "datetimeoffset_types";
+
+            using JsonDocument actualJsonDoc = JsonDocument.Parse(actual);
+            using JsonDocument expectedJsonDoc = JsonDocument.Parse(expected);
+
+            string actualDateTimeOffset = actualJsonDoc.RootElement.GetProperty(fieldName).ToString();
+            string expectedDateTimeOffset = expectedJsonDoc.RootElement.GetProperty(fieldName).ToString();
+
+            // handles cases when one of the values is null
+            if (string.IsNullOrEmpty(actualDateTimeOffset) || string.IsNullOrEmpty(expectedDateTimeOffset))
+            {
+                Assert.AreEqual(expectedDateTimeOffset, actualDateTimeOffset);
+            }
+            else
+            {
+                Assert.AreEqual(DateTimeOffset.Parse(expectedDateTimeOffset), DateTimeOffset.Parse(actualDateTimeOffset));
+            }
+        }
+
+        /// <summary>
+        /// Required due to different format between SQL time and HotChocolate TimeSpan time(ISO-8601) result.
+        /// </summary>
+        private static void CompareTimeResults(string actual, string expected)
+        {
+            string fieldName = "time_types";
+
+            using JsonDocument actualJsonDoc = JsonDocument.Parse(actual);
+            using JsonDocument expectedJsonDoc = JsonDocument.Parse(expected);
+
+            if (actualJsonDoc.RootElement.ValueKind is JsonValueKind.Array)
+            {
+                ValidateArrayResults(actualJsonDoc, expectedJsonDoc, fieldName);
+                return;
+            }
+
+            string actualTimeString = actualJsonDoc.RootElement.GetProperty(fieldName).ToString();
+            string expectedTimeString = expectedJsonDoc.RootElement.GetProperty(fieldName).ToString();
+
+            // handles cases when one of the values is null
+            if (string.IsNullOrEmpty(actualTimeString) || string.IsNullOrEmpty(expectedTimeString))
+            {
+                Assert.AreEqual(expectedTimeString, actualTimeString);
+            }
+            else
+            {
+                TimeOnly actualTime = TimeOnly.Parse(actualTimeString, CultureInfo.InvariantCulture);
```

Comment:
can you add comments why you chose certain `CultureInfo.` values? What kind of documentation do we need to add to Dab docs to clarify what timezone/lack of timezone / culture may be expected in GraphQL results and request payloads?